codex-rs/tui/src/bottom_pane/mod.rs

```diff
@@ -121,49 +125,64 @@ impl BottomPane {
     }
 
     pub fn desired_height(&self, width: u16) -> u16 {
-        // Always reserve one blank row above the pane for visual spacing.
         let top_margin = 1;
 
         // Base height depends on whether a modal/overlay is active.
         let base = match self.active_view().as_ref() {
             Some(view) => view.desired_height(width),
-            None => self.composer.desired_height(width).saturating_add(
-                self.status
+            None => {
+                let status_height = self
+                    .status
                     .as_ref()
-                    .map_or(0, |status| status.desired_height(width)),
-            ),
+                    .map_or(0, |status| status.desired_height(width));
```

Comment:
Option<Renderable> implements Renderable, so this might be able to just be 

```suggestion
                let status_height = self.status.desired_height(width);
```

Or something similar (may need the as_ref() call)